templates/function-template-nodejs/index.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
For consistency between the templates w.r.to naming, we should call this as `azure-function-template-nodejs`. Thoughts?

Reply:
but the aws template does not say aws but lambda-template-nodejs

Reply:
aws doesn't follow it since the start (& we can't change that in this PR!). So, was thinking of following the convention for azure templates.

Reply:
ok I will change the file name